mcsema/Arch/ABI.cpp

```diff
@@ -670,13 +674,33 @@ void CallingConvention::StoreArguments(
     // Bump the stack pointer.
     alloc_size = std::max<uint64_t>(alloc_size, addr_size);
     sp = ir.CreateSub(sp, llvm::ConstantInt::get(gWordType, alloc_size));
+    num_stored_stack_bytes += alloc_size;
   }
```

Comment:
Would it be good to have a final `else { LOG(ERROR) << "Unrecognized calling convention"; }`?

Reply:
At that line or a different place?

Reply:
I must have misread the code diff. There are already `LOG(FATAL)` outputs in the places I meant. They're not explicit `else` blocks, but they do immediately follow the `if-else-if` blocks.